skins/laika/src/components/pages/donation_form/subpages/AddressPage.vue

```diff
@@ -134,13 +139,18 @@ export default Vue.extend( {
 							document.getElementsByClassName( 'help is-danger' )[ 0 ].scrollIntoView( { behavior: 'smooth', block: 'center', inline: 'nearest' } );
 							return;
 						}
-						this.$emit( 'submit-donation' );
+						( this as any ).submitDonationForm();
```

Comment:
😭 😞 
Nothing you can do here with our current setup, I'm really looking forward to Vue 3

Reply:
Yeah, I was surprised the function is not recognized by TS.
I know we have other cases like this and there's no TS complaint there, idk :woman_shrugging: